specification/dartLangSpec.tex

```diff
@@ -7427,7 +7428,7 @@ \subsection{Sets}
 \item
 For each $i \in 1 .. n$ in numeric order,
 the expression $e_i$ is evaluated producing object $v_i$.
-\item A fresh object (\ref{generativeConstructors}) $s$
+\item A fresh instance (\ref{generativeConstructors}) $s$
```

Comment:
Reason for change: This line is the only occurrence of `fresh object`, it's `fresh instance` everywhere else.